### PR TITLE
[CLI] Add Spender encoding commands

### PR DESCRIPTION
**Pull request by ryangoree**

### CODE REVIEW

File: packages/council-cli/src/commands/encode/spender/high-spend.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      const destination = await requiredString(args.destination, {
+        name: "destination",
+        message: "Enter destination address",
+      });
+
+      signale.success(encodeSmallSpend(amount, decimals, destination));
+    },
+  });
+
+export function encodeSmallSpend(
```
Comment (DannyDelott) [inline packages/council-cli/src/commands/encode/spender/high-spend.ts:56]: typo: HighSpend

File: packages/council-cli/src/commands/encode/spender/medium-spend.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      const destination = await requiredString(args.destination, {
+        name: "destination",
+        message: "Enter destination address",
+      });
+
+      signale.success(encodeSmallSpend(amount, decimals, destination));
+    },
+  });
+
+export function encodeSmallSpend(
```
Comment (DannyDelott) [inline packages/council-cli/src/commands/encode/spender/medium-spend.ts:56]: here too

**Review by DannyDelott — APPROVED** [review]

File: packages/council-cli/src/commands/encode/spender/high-spend.ts (hunk 1 above)
Comment (ryangoree) [reply packages/council-cli/src/commands/encode/spender/high-spend.ts:56]: 🙏

File: packages/council-cli/src/commands/encode/spender/medium-spend.ts (hunk 2 above)
Comment (ryangoree) [reply packages/council-cli/src/commands/encode/spender/medium-spend.ts:56]: 🙏

**Pull request merged**